src/boot/api.ts

```diff
@@ -14,6 +18,7 @@ import Web3 from 'web3';
 
 const $api = ref<ApiPromise>();
 const $web3 = ref<Web3>();
+const $isEnableIndividualClaim = ref<boolean>(false);
```

Comment:
We have a module function check function https://github.com/AstarNetwork/astar-apps/blob/13e951fe5214a485448848eb1f4fb002ca6a7b39/src/hooks/useStakingH160.ts#L196 from the MetaMask dApps Staking PR. Having the module check spread around the code will make things difficult to maintain.